metricbeat/module/docker/cpu/cpu_test.go

```diff
@@ -73,6 +74,8 @@ func TestCPUService_UsageInUsermode(t *testing.T) {
 	//  THEN
 	assert.Equal(t, float64(0.50), result)
 }
+
+/* TODO: uncomment
```

Comment:
Should I comment it? This function needs to be updated in order to match with last changes you made. I'll do it.